udata_tabular_preview/preview.py

```diff
@@ -28,7 +29,8 @@ def can_preview(self, resource):
         is_allowed = allow_remote or not is_remote
 
         max_size = current_app.config.get('TABULAR_MAX_SIZE')
-        extras_size = resource.extras.get('check:headers:content-length')
+        extras_size = resource.extras.get('analysis:filesize') \
```

Comment:
Following https://github.com/etalab/udata-hydra/pull/66
```suggestion
        extras_size = resource.extras.get('analysis:content-length') \
```